src/shared/Microsoft.Git.CredentialManager/Microsoft.Git.CredentialManager.csproj

```diff
@@ -7,10 +7,15 @@
     <RootNamespace>Microsoft.Git.CredentialManager</RootNamespace>
     <IsTestProject>false</IsTestProject>
     <LangVersion>latest</LangVersion>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net461'">
     <Reference Include="System.Net.Http" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="LibGit2Sharp.NativeBinaries" Version="2.0.278" />
```

Comment:
@jrbriggs we'll want to clear this dependency with compliance.
We may need to include the license file and a copy of the source?